tests/tests/cpld_control_sfs_180d

```diff
@@ -0,0 +1,116 @@
+#
+#  cpld_control GFSv17 with UGWPv1 test, i.e., GSL orographic drag +
+#  version 1 non-stationary GWD, hydrostatic, long forecast
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system hydrostatic version - C96MX100 - long forecast"
+
+export CNTL_DIR=cpld_control_sfs_180d
+
+export LIST_FILES="sfcf8280.nc \
+                   sfcf8640.nc \
+                   atmf8280.nc \
+                   atmf8640.nc \
+                   GFSFLX.GrbF8280 \
+                   GFSFLX.GrbF8640 \
+                   GFSPRS.GrbF8280 \
+                   GFSPRS.Grb8640 \
+                   RESTART/20220317.060000.MOM.res.nc" 
+
+export_fv3
+export_cpl
+export_ugwpv1
```

Comment:
This should be fine, but as of #2425 `export_cpl` should come after `export_ugwpv1` to set the variables as intended. (Idea is to set atm vars first then add coupling if needed).

Reply:
Done.